.github/workflows/ci.yml

```diff
@@ -493,7 +494,10 @@ jobs:
       - go-apidiff
       - docker-checks
     steps:
-      - run: exit 0
+      - if: failure() || cancelled()
+        run: exit 1
```

Comment:
:question: Does this detect if one of the jobs above was skipped? In that case, which part is true? `failed()`? `cancelled()`?

Reply:
No I don't think it detects it and based on https://github.com/camunda/zeebe/issues/16458#issuecomment-1952912657, I'd think that the `success()` branch applies.